python/triton/compiler/compiler.py

```diff
@@ -237,8 +237,8 @@ def compile(src, target=None, options=None):
     # core changes to make it easier to track kernels by hash.
     enable_override = os.environ.get("TRITON_KERNEL_OVERRIDE", "0") == "1"
     enable_ir_dump = os.environ.get("TRITON_KERNEL_DUMP", "0") == "1"
-    fn_override_manager = get_override_manager(src.hash()) if enable_override else None
-    fn_dump_manager = get_dump_manager(src.hash()) if enable_ir_dump else None
+    fn_override_manager = get_override_manager(hash) if enable_override else None
+    fn_dump_manager = get_dump_manager(hash) if enable_ir_dump else None
```

Comment:
we don't use hash on purpose so that the hash doesn't change when we rebuild triton (see comment above)

Reply:
@ThomasRaoux, sorry, totally missed the comment. I will close this PR and figure out what's causing my issue of the override directory not being picked up without this change.

Reply:
no worries, maybe it should be made more explicit as this is not the first time someone try to change it. We should definitely have a readme for overriding